packages/shopify-codemod/transforms/global-assignment-to-default-export.js

```diff
@@ -1,6 +1,24 @@
 import {findFirstMember, insertAfterDirectives} from './utils';
 
 export default function globalAssignmentToDefaultExport({source}, {jscodeshift: j}, {printOptions = {}, appGlobalIdentifiers}) {
+  function removeFirstMember(memberExpression) {
```

Comment:
👍 Really nice!  Updated to use that.

The computed `second` scenario won't come up because `isDefaultExportMemberAssignment` is comparing stringified paths that contain a `.`.  If this function was in utils, I'd do it, but I'm going to stay scrappy here.